canonical/snapcraft.io
# Fetch, display and create models

## Done
- Fetch, display and filter models
- Create new model

## How to QA
- Go to https://snapcraft-io-4340.demos.haus/login-beta and login
- Go to https://snapcraft-io-4340.demos.haus/admin/njwQYXFnS7ppo7LaGxoh7aqVZc1CPi26/models
- Check that models are loaded onto to the page
- Check that you can filter models
- Check that reloading the page with a filter applied works
- Check that you can create a new model (this will fail if you don't provide an API key but there is [an issue to fix this](https://warthogs.atlassian.net/browse/WD-5151)

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-4997
